rbac/api/cross_access/view.py

```diff
@@ -260,3 +258,60 @@ def update_status(self, car, status):
         if status == "approved":
             create_cross_principal(car.target_account, car.user_id)
         car.save()
+
+    def check_patch_permission(self, request, update_obj):
+        """Check if user has right to patch cross access request."""
+        if request.user.account == update_obj.target_account:
+            """ For approvers updating requests coming to them, only org admin
+                could update status from pending/approved/denied to approved/denied.
+            """
+            if not request.user.admin:
+                self.throw_validation_error("cross-account partial update", "Only org admin could update status.")
```

Comment:
```suggestion
                self.throw_validation_error("cross-account partial update", "Only org admins may update the request status.")
```

Reply:
Thanks for the suggestions! Will commit them in my next version